website/content/en/docs/reference/threat-model.md

```diff
@@ -29,92 +28,161 @@ A Cluster Developer has the ability to create pods via `Deployments`, `ReplicaSe
+
+#### Karpenter can orchestrate the creation/deletion of IAM Instance Profiles it doesn’t own
+
+**Threat:** Karpenter has permission to create/update/delete instance profiles as part of its controller permissions to ensure that it can auto-generate instance profiles when `EC2NodeClasses` are created. An actor who obtains control of the Karpenter pod’s IAM role may delete instance profiles not owned by Karpenter, causing workload disruption to other instances using the profile in the account.
+
+**Mitigation:** Karpenter’s controller permissions enforce that it creates instance profiles with tags which provide ownership information. These tags include:
 
 * `karpenter.sh/managed-by`
 * `kubernetes.io/cluster/${CLUSTER_NAME}`
 * `karpenter.k8s.aws/ec2nodeclass`
 * `topology.kubernetes.io/region`
 
-These tags ensure that instance profiles created by Karpenter in the account are unique to that cluster. Karpenter's controller permissions _only_ allow it to act on instance profiles that contain these tags which match the cluster information.
+Source: https://karpenter.sh/docs/reference/cloudformation/#allowscopedinstanceprofilecreationactions
+
+These tags ensure that instance profiles created by Karpenter in the account are unique to that cluster. Karpenter’s controller permissions only allow it to act on instance profiles that contain these tags which match the cluster information.
+
+#### Operator and developer have different permissions
+
+**Threat:** When installing Karpenter, cluster operators define an [IAM Role](https://karpenter.sh/docs/reference/cloudformation/#controller-authorization) for Karpenter to run under. Additionally, cluster operators must provide an [Instance IAM Role](https://karpenter.sh/docs/reference/cloudformation/#node-authorization) or Instance Profile for EC2 Instance launched by Karpenter. If the developer had permission to exec into the Karpenter pod, they would elevate their permissions to that of the IAM role for Karpenter.
+
+**Mitigation:** Karpenter runs in a separate namespace [(currently recommended to run under kube-system namespace)](https://karpenter.sh/docs/getting-started/getting-started-with-karpenter/#preventing-apiserver-request-throttling) , which enables cluster operators to restrict access using Kubernetes RBAC. 
+
+#### Undesired Disruption Through NodePool Access
+
+Problem: If the user is able to modify the `NodePool` for a given budget, they can edit the budgets to change the resulting behavior. This could either terminate nodes when the user doesn’t want it, or block Karpenter to terminate and patch nodes when a user does. 
```

Comment:
```suggestion
Problem: If the user is able to modify the `NodePool`, they can edit the budgets to allow the disruption time windows to be more or less permissive than the user intended. This could either terminate nodes when the user doesn’t want it or block Karpenter to terminate and patch nodes when a user does. 
```